java/test/jmri/jmrit/logixng/actions/ActionThrottleTest.java

```diff
@@ -330,46 +338,50 @@ public void testCtorAndSetup3() {
 //        Assert.assertEquals("child is correct bean",
 //                childSocket0,
 //                expression.getChild(0).getConnectedSocket());
-        Assert.assertEquals("expression has 3 female sockets", 3, expression.getChildCount());
+        Assert.assertEquals("expression has 5 female sockets", 5, expression.getChildCount());
 
         Assert.assertTrue("expression female socket is connected",
                 expression.getChild(1).isConnected());
 //        Assert.assertEquals("child is correct bean",
 //                childSocket1,
 //                expression.getChild(1).getConnectedSocket());
-        Assert.assertEquals("expression has 3 female sockets", 3, expression.getChildCount());
+        Assert.assertEquals("expression has 5 female sockets", 5, expression.getChildCount());
 
         Assert.assertTrue("expression female socket is connected",
                 expression.getChild(2).isConnected());
 //        Assert.assertEquals("child is correct bean",
 //                childSocket2,
 //                expression.getChild(2).getConnectedSocket());
 
-        Assert.assertEquals("expression has 3 female sockets", 3, expression.getChildCount());
+        Assert.assertEquals("expression has 5 female sockets", 5, expression.getChildCount());
 
         // Try run setup() again. That should not cause any problems.
         expression.setup();
 
-        Assert.assertEquals("expression has 3 female sockets", 3, expression.getChildCount());
+        Assert.assertEquals("expression has 5 female sockets", 5, expression.getChildCount());
     }
 
     @Test
     public void testGetChild() {
-        Assert.assertTrue("getChildCount() returns 3", 3 == actionThrottle.getChildCount());
+        Assert.assertTrue("getChildCount() returns 3", 5 == actionThrottle.getChildCount());
 
         Assert.assertNotNull("getChild(0) returns a non null value",
                 actionThrottle.getChild(0));
         Assert.assertNotNull("getChild(1) returns a non null value",
                 actionThrottle.getChild(1));
         Assert.assertNotNull("getChild(2) returns a non null value",
                 actionThrottle.getChild(2));
+        Assert.assertNotNull("getChild(3) returns a non null value",
+                actionThrottle.getChild(3));
+        Assert.assertNotNull("getChild(4) returns a non null value",
+                actionThrottle.getChild(4));
 
         boolean hasThrown = false;
         try {
-            actionThrottle.getChild(3);
+            actionThrottle.getChild(5);
```

Comment:
Are all these "5" values really NUM_LOCO_SOCKETS?

Reply:
No need to change it on my account.  I'd probably have put NUM_LOCO_SOCKETS in, with a comment "should assert if >= NUM_LOCO_SOCKETS" or similar. But reasonable people can do this in multiple ways.